kolibri/plugins/management/assets/src/views/manage-class-page/index.vue

```diff
@@ -2,11 +2,14 @@
 
   <div>
 
-    <div class="header">
-      <h1>{{ $tr('allClasses') }}</h1>
+    <div 
+      class="header"
```

Comment:
How will this class and the dynamic class interact?

If the intention is that the dynamic class overwrites the non-dynamic class, it might be better to do the following:

`:class="{ 'header-mobile': windowSize.breakpoint <= 0, 'header': windowSize.breakpoint > 0}"`

Reply:
You can also use multiple classes to increase the specificity and explicitness of your overrides. For example, I might do something like this:

```html
<template>
  <div class="message" :class="{'mobile': windowSize.breakpoint <= 1}">
    Hello World
  </div>
</template>

<style scoped>

  .message
    position: absolute;
    top: 0;
    left: 0;
    font-size: 18px

  .message.mobile
    font-size: 14px

</style>
```

however it looks like in this case, adding `.header-mobile` doesn't actually do anything!

```stylus
  .header
    position: relative
    padding-right: 150px
    margin-bottom: 16px

  .header-mobile
    position: relative
```